packages/sources/react/src/components/structure/VtmnList/VtmnList.tsx

```diff
@@ -33,6 +33,35 @@ export const VtmnListItemEndAction = ({
   </div>
 );
 
+/**
+ * Custom function to inject elements under the link
+ * NB: Outside the link must be only the VtmnListItemEndAction because it can be a action button
+ * @param children
+ * @param className
+ * @param target
+ * @param props
+ * @constructor
+ */
+export const VtmnListItemLink = ({
+  children,
+  className,
+  target,
+  ...props
+}: React.ComponentPropsWithoutRef<'a'>) => (
+  <>
+    <a className={clsx('vtmn-list__link', className)} {...props}>
+      {React.Children.toArray(children).filter(
+        (child) =>
+          React.isValidElement(child) && child.type !== VtmnListItemEndAction,
+      )}
+    </a>
+    {React.Children.toArray(children).filter(
+      (child) =>
+        React.isValidElement(child) && child.type === VtmnListItemEndAction,
+    )}
+  </>
+);
+
```

Comment:
Could it be necessary to add an early return with : 
```
if (!React.Children) return <></>
```
?
What happens here if there is no React.Children ?

Reply:
This behaviour doesn't exist on other langages.
In fact, the component should not worked if we don't put the text slot (VtmnListItemText)